operators/README.md

```diff
@@ -73,8 +75,15 @@ The Instance Operator implements the backend logic necessary to spawn new enviro
 ### Persistent Feature
 The Instance Operator allows you to create persistent Virtual Machines. Persistent means that the VM can be stopped and restarted, deleted and recreated without data loss. To make this possible, the content of the VM's disk is stored in a PVC.
 In order to use this feature, [the CDI operator](../infrastructure/virtualization/README.md) has to be installed.
-Granted the CDI operator has been deployed, the Instance Operator spawns a persistent VM similarly to a normal (i.e. non-persistent) one. To this end, the selection about which version to create is performed looking at the corresponding field in the `template` spec.
-N.B. The process of creating a persistent VirtualMachine can take a bit more time with the respect to a normal one (5-10 mins, depending on the size of the image). However when you recreate the VM you will not have to wait such that time.
+
+Granted the CDI Operator has been deployed, the Instance Operator spawns a persistent VM similarly to a normal (i.e. non-persistent) one. To this end, the selection about which version to create is performed looking at the corresponding field in the `template` spec. It is also possible to select the amount of memory needed for the VM by modifying the corresponding `template` spec.
+
+It is important to notice that the Instance Operator has to watch not only to the instance but also the datatvolume associated to it.
+Once the instance is created, the controller starts creating a datavolume, whose creation triggers the CDI controller which creates two resources: the PVC and the importer pod.
+
+The aim of the importer pod is to exctract an image from the docker registry and load it inside the PVC. This process, depending on the size of the image, can take some minutes. Once the import is completed the Instance Operator wakes up (this is the only one case in which a datavolume resource is able to to trigger a reconcile, thanks to the filter and predicates used) and all other resources are created: service, secret, ingress, oauth2 ones and VM (and VMI in case the instance running field is set to true).
```

Comment:
```suggestion
The aim of the importer pod is to extract an image from the docker registry and load it inside the PVC. This process, depending on the size of the image, can take some minutes. Once the import is completed, the Instance Operator wakes up (this is the only case in which a DataVolume resource modification triggers a reconcile, thanks to the filter and predicates used) and all the other resources are created as described in the following.
```

Please:
* Double check for typos and be consistent with the rest of the text;
* Introduce concepts that may not be clear (I doubt a newbie reader would know what you mean by filter and predicate, you should at least add a link to the code and then describe a little the approach). Also the difference between VirtualMachine and VirtualMachineInstance deserves to be explained more in detail, as it is a key aspect of the design. Which docker registry are you referring to?
* Do not use abbreviations if not previously introduced.